src/config/field-overrides/programNotificationTemplate.js

```diff
@@ -56,6 +56,7 @@ export default new Map([
     }],
     ['recipientUserGroup', {
         component: (props) => {
+            console.log(props);
```

Comment:
👆 Is this a development log line? Either remove it before merging to master or replace it with the log module used for logging elsewhere in the app.

Reply:
Ah good catch, that was just for temporary debugging